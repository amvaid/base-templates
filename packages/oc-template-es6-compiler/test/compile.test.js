jasmine.DEFAULT_TIMEOUT_INTERVAL = 100000;
test('', () => {});
// WIP
