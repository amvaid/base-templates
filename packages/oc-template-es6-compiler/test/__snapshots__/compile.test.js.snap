// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should correctly compile the oc component 1`] = `
Object {
  "dependencies": Object {
    "lodash": "",
  },
  "description": "",
  "devDependencies": Object {
    "oc-template-es6-compiler": "",
  },
  "name": "es6-component",
  "oc": Object {
    "date": "",
    "files": Object {
      "dataProvider": Object {
        "hashKey": "8d9816a8b8c36a5a86d41d2dbbb22624425b1979",
        "src": "server.js",
        "type": "node.js",
      },
      "static": Array [],
      "template": Object {
        "hashKey": "cc6968375e678d61ca630a83c7eddceeb15385bd",
        "src": "template.js",
        "type": "oc-template-es6",
        "version": "",
      },
    },
    "packaged": true,
    "version": "1.0.0",
  },
  "repository": "",
  "version": "1.0.0",
}
`;

exports[`Should correctly compile the oc component 2`] = `
Array [
  "../../../mocks/es6-component/__package/package.json",
  "../../../mocks/es6-component/__package/server.js",
  "../../../mocks/es6-component/__package/styles.css",
  "../../../mocks/es6-component/__package/template.js",
]
`;
